lxdock/provisioners/ansible.py

```diff
@@ -16,13 +16,13 @@ class AnsibleProvisioner(Provisioner):
     name = 'ansible'
 
     guest_required_packages_alpine = ['openssh', 'python', ]
-    guest_required_packages_archlinux = ['openssh', 'python2', ]
+    guest_required_packages_arch = ['openssh', 'python', ]
```

Comment:
Under arch, `python` refers to python 3. Ansible requires (unless you're bleeding edge and want to use the newly added py3 support in ansible) python2.